tests/integration/states/pip.py

```diff
@@ -361,7 +364,6 @@ def test_issue_6833_pip_upgrade_pip(self):
             # previously installed
             ret = self.run_function(
                 'pip.install', ['pip==6.0'], upgrade=True,
-                ignore_installed=True,
```

Comment:
I just don't think removing an argument altogether that has been part of this test for a while already on other platforms, just to get it passing on another, is a good move. Let's work in Slack to figure out what was happening and perhaps change this line to something like ``ignore_installed=True if grains['os'] not in ('MacOS',) else False``